index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
```

Comment:
The problem here is that too many Tetris pieces are the same color. Add more colors so it can be easier to see :)